submodules/usersCallForwarding/usersCallForwarding.scss

```
@@ -0,0 +1,190 @@
+.call-forwarding-popup {
+    width: 976px;
+}
+
+.feature-popup-title {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.feature-popup-flex {
+    flex-direction: column;
+}
+
+.call-forwarding-titles {
+    color: rgb(14, 14, 15);
+    font-size: 16px;
+    font-weight: normal;
+    height: 24px;
+    line-height: 24px;
+}
+
+.call-forwarding-descriptions {
+    color: rgb(14, 14, 15);
+    font-size: 14px;
+    font-weight: normal;
+    height: 20px;
+    line-height: 20px;
+
+}
+
+.popup-container .feature-popup-title {
+    display: flex;
+    align-items: center;
+}
+
+.simple-strategy-row {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.simple-strategy {
+    display: block;
+    width: 95%;
+}
+
+.disabled {
+    display: none;
+}
+
+.controlls-width {
+    background: rgb(255, 255, 255);
```

For example, after importing the `_base.scss`, you do the following here:
```suggestion
    background: $white;
```